experiments/mp-game-6/draw.creatures.js

```diff
@@ -116,20 +161,46 @@ function drawTestCreature(stim, creatureInd, scale, roundProps){
             markAsSpecies(id);
             roundProps.selected_test_stim.push(id);
         }
-        console.log(roundProps.selected_test_stim)
     });
 }
   
 function darken(id) {
-    $(id).css({"opacity": 1.0});
+    $(id).css({"opacity":1.0});
+}
+function lighten(id){
+    $(id).css({"opacity":.5});
 }
   
-function showSpeciesIndicator(id, speciesName) {
-    var labelID = id + "_label";
-    $(labelID).css("visibility", "visible");
-  
-    if ($(id).attr("belongs_to_concept") === "true")
-        $(id).css({"border":"2px dashed black"});
+function turnLightsOn(buttonsOn, test1, numButtons, numLights, lightsPrev) {
+    var lightsOn = [];
+    var arrOfButtons = [];
+    for(var i = 1; i<=numButtons; i++){
```

Comment:
Related to above comment - that way you don't have to do this searching through buttons pressed to create a fixed length array `[true, false, true]` - your selectedTestStims can just be that fixed length array in the first place

Reply:
You fixed this in the second commit.